pkg/constants/constants.go

```diff
@@ -0,0 +1,15 @@
+package constants
+
+const (
+	// KlusterletDeployModeAnnotation is the annotation key of klusterlet deploy mode, it describes the
+	// klusterlet deploy mode when importing a managed cluster.
+	// If the value is "Hosted", the HostingClusterNameAnnotation annotation will be required, we use
+	// AnnotationKlusterletHostingClusterName to determine where to deploy the registration-agent and
+	// work-agent.
+	AnnotationKlusterletDeployMode string = "import.open-cluster-management.io/klusterlet-deploy-mode"
```

Comment:
suggest ot move the constants to clc-api repo https://github.com/stolostron/cluster-lifecycle-api .
it could be more easy to manage and use across the repos.